testutil/logging.go

```diff
@@ -0,0 +1,22 @@
+package testutil
```

Comment:
Nits. Copyright notice for the new file.

Reply:
I wonder why the license test didn't catch this?